src/github/tasksDashboard.ts

```diff
@@ -0,0 +1,155 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { Disposable, disposeAll } from '../common/lifecycle';
+import { ITelemetry } from '../common/telemetry';
+import { CopilotRemoteAgentManager } from './copilotRemoteAgent';
+import { RepositoriesManager } from './repositoriesManager';
+import { DashboardWebviewProvider } from './taskDashboardWebviewProvider';
+
+export class TasksDashboardManager extends Disposable implements vscode.WebviewPanelSerializer {
+	public static readonly viewType = 'github-pull-request.projectTasksDashboard';
+	private static readonly viewTitle = vscode.l10n.t('Tasks Dashboard');
+
+	private _currentView: {
+		readonly webview: vscode.WebviewPanel;
+		readonly dashboardProvider: DashboardWebviewProvider;
+		readonly disposables: vscode.Disposable[];
+	} | undefined;
+
+	private _statusBarItem?: vscode.StatusBarItem;
+
+	constructor(
+		private readonly _context: vscode.ExtensionContext,
+		private readonly _repositoriesManager: RepositoriesManager,
+		private readonly _copilotRemoteAgentManager: CopilotRemoteAgentManager,
+		private readonly _telemetry: ITelemetry
+	) {
+		super();
+
+		// Create status bar item for launching dashboard
+		this.updateStatusBarItem();
+
+		this._register(vscode.workspace.onDidChangeConfiguration(e => {
+			if (e.affectsConfiguration('githubPullRequests.projectTasksDashboard.enabled')) {
+				this.updateStatusBarItem();
+			}
+		}));
+
+		// Register webview panel serializer for tasks dashboard
+		this._register(vscode.window.registerWebviewPanelSerializer(TasksDashboardManager.viewType, this));
+	}
+
+	public override dispose() {
+		super.dispose();
+
+		this._currentView?.disposables.forEach(d => d.dispose());
+		this._currentView = undefined;
+
+		this._statusBarItem?.dispose();
+		this._statusBarItem = undefined;
+	}
+
+	private updateStatusBarItem(): void {
+		const dashboardEnabled = vscode.workspace.getConfiguration('githubPullRequests')
+			.get<boolean>('projectTasksDashboard.enabled', false);
+
+		if (dashboardEnabled && !this._statusBarItem) {
+			// Create status bar item if it doesn't exist and is now enabled
+			this._statusBarItem = this._register(vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left, 100));
+			this._statusBarItem.text = '$(dashboard) Tasks';
```

Comment:
The text should also be localized so that the order of the icon and word are correct in RTL languages.